db/migrate/20221215120706_remove_description_from_users.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveDescriptionFromUsers < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :users, :description, :string
+  end
+end
```

Comment:
- [ ] I had this issue setting up your project.

  ![udid](https://user-images.githubusercontent.com/81468675/208047426-c3d5b4a9-9085-4eac-901d-4614b489eb5b.jpg)

  You are trying to remove a column that never existed. From your migration history, I can see there was no point at which you added `description` to the users table, so it throws errors when trying to remove a non-existent column. When I removed the file everything worked just fine. Kindly delete this migration file.